src/components/fields/SingleSelect/Settings.tsx

```diff
@@ -42,7 +114,18 @@ export default function Settings({ onChange, config }: ISettingsProps) {
           marginBottom: 5,
         }}
       >
-        {options?.map((option: string) => (
+        <DndProvider backend={HTML5Backend}>
+          {options?.map((option: string, index: number) => (
+            <DraggableSettingsCard
+              options={options}
+              option={option}
+              index={index}
+              onChange={onChange}
+            />
+          ))}
+        </DndProvider>
+
+        {/* {options?.map((option: string) => (
```

Comment:
you can remove the old code here, instead of commenting it out

Reply:
@shamsmosowi  Ok let me check  and update it

Reply:
@shamsmosowi  fixed the above mentions, kindly check and let me know if everything is needed

Reply:
Hi @saravanan-inc , i see you've removed the commented code, but the interaction highlighted in the video is still not updated

Reply:
Hi @shamsmosowi the interaction in too updated could you please elaborate the issue ?

Reply:
The draggable icon and the animation are already updated in the previous commit, please check the code.